draft-ietf-rtcweb-jsep.xml

```diff
@@ -2359,20 +2359,14 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             be generated according to the same rules as for an initial
             offer.</t>
 
-            <t>Each "m=" and c=" line MUST be filled in with the port,
-            protocol, and address of the default candidate for the m=
+            <t>Each "m=" and c=" line MUST be filled in with the port, profile,
```

Comment:
I've just noticed that this document refers to SDP protocol as profile. If you look at the references for the documents that define "profiles" in JSEP, they all refer to them as protocols. For SRTP profile is actually something different and refers to SRTP key parameters.

Also, default candidate does not have profile (or transport), it has transport, port, and (connection-)address. So each "m=" and "c=" MUST be filled in with port, protocol, and address to match port, transport, and address of the default candidate for the m= section, as described in ice-sip.sdp.

Reply:
It's kind of ambiguous; https://tools.ietf.org/html/rfc7850 uses "profile", "proto", and "protocol". 

Overall this change would extend outside of this specific block of text; as such, I think you should raise it on the mailing list and we can consider it outside of the scope of this review.

Reply:
I will take the first comment to the list.

The second comment still applies: Default candidate does not have a profile (or protocol) as in UDP/TLS/SRTP/AVPF. It has transport (UDP or TCP), port and address.  What we are trying to say is that profile in m= line must use the same transport as default candidate, but we cannot say that m= line must use the profile from default candidate, since no profile is specified in the candidate.

Reply:
OK, that makes sense, will adjust accordingly.

Reply:
I think I addressed the issue. PTAL.

Reply:
This looks good.